apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -48,6 +89,36 @@ class CoursePaneButtonRow extends PureComponent<CoursePaneButtonRowProps> {
                         <Refresh />
                     </IconButton>
                 </Tooltip>
+
+                <Tooltip title="Hide Columns">
+                    <IconButton onClick={this.handleClick} className={classes.button}>
+                        <MoreVert />
+                    </IconButton>
+                </Tooltip>
+
+                {this.state.columnSelector && (
+                    <FormControl className={classes.formControl}>
+                        <Select
+                            style={{ width: '0', zIndex: -1, color: isDarkMode() ? '#303030' : '#FAFAFA' }}
+                            inputProps={{ IconComponent: () => null }} // some styling nonsense to Thanos snap away the little arrow icon: https://aguidehub.com/blog/2023-02-25-how-to-hide-arrow-from-mui-select-in-react-js/
+                            open={this.state.columnSelector}
+                            multiple
+                            onClose={this.handleClick}
+                            value={this.state.activeColumns}
+                            onChange={this.handleChange}
+                        >
+                            {columnList.map((column) => (
+                                <MenuItem key={column.value} value={column.value} style={{ maxWidth: '200px' }}>
+                                    <Checkbox
+                                        checked={this.state.activeColumns.indexOf(column.value) >= 0}
+                                        color="default"
+                                    />
+                                    <ListItemText primary={column.label} />
+                                </MenuItem>
+                            ))}
+                        </Select>
+                    </FormControl>
```

Comment:
I would extract this to be a separate component so that it has its own name so that its function would be immediately obvious